FrostyPlugin/App.cs

```diff
@@ -19,6 +20,8 @@ public sealed class App
         public static PluginManager PluginManager;
         public static NotificationManager NotificationManager;
 
+        public static List<int> WhitelistBundles = new List<int>();
```

Comment:
I would rename this to "WhitelistedBundles"

Reply:
updated